docs/development/mono-repository.md

```diff
@@ -0,0 +1,95 @@
+---
+sidebar_position: 7
+---
+
+# Mono-Repository (monorepo) Tooling and Guidance
+
+Working across multiple repositories for the same project and/or team is cumbersome and slows down developer velocity. A way to improve developer velocity for teams and projects is to leverage a mono-repository (monorepo). With a monorepo, teams will be able to maintain multiple packages in a singular repository without having to switch to different repositories to maintain other packages.
+
+## Tooling
+
+To enable a mono-repository, we recommend leveraging [yarn workspaces][yarn-workspaces] or [npm workspaces][npm-workspaces].
+
+### Setup with Yarn Example
+
+#### Modify package.json in the workspace root
+
+```json
+{
+  "private": true,
+  // this is an array containing glob paths to each workspace
+  "workspaces": ["workspace-a", "packages/*"]
+}
+```
+
+#### Create subfolders and package.json for each package in the monorepo
+
+```json
+// /workspace-a/package.json
+{
+  "name": "workspace-a",
+  "version": "1.0.0",
+  "dependencies": {
+    "foo": "0.0.1"
+  }
+}
+
+// /packages/foo/package.json
+{
+  "name": "foo",
+  "version": "1.0.0"
+}
+```
+
+#### Run `yarn install` in the workspace root
+
+
+Given the above, we should expect the following folder structure
+```
+/package.json
+/yarn.lock
+/workspace-a/package.json
+/packages/foo/package.json
+/node_modules
+/node_modules/workspace-a -> /workspace-a
+```
+
+#### Maintaining
+
+With a monorepo, teams can now leverage a common set of tooling for [linting][linting] and [testing][testing] across all packages in the repository. See our [code-consistency][code-consistency] section for further guidance.
```

Comment:
* Rephrase `teams can now leverage` to `it is recommended to leverage`. 
* Add on rebutal statement `however teams can also control scripts individually per package`
* Provide example on individually controlled script packages
   * Provide use case when you might have scripts that need to be defined differently due to different package behaviors